src/renderer/themes.css

```diff
@@ -444,6 +468,61 @@ it can be safely elided. This looks quite pleasant on this theme. */
   --primary-color-active: #D84315;
 }
 
+.mainGruvboxGreen,
+.mainGruvboxRed,
+.mainGruvboxYellow,
+.mainGruvboxBlue,
+.mainGruvboxPurple,
+.mainGruvboxAqua,
+.mainGruvboxOrange {
+    --text-with-main-color: #282828;    
```

Comment:
This `text-with-main-color`, and the other `--text-with-accent-color`, (which should also both be the same color) are still at a notably low color contrast with `mainGruvboxRed`  and `secGruvboxRed`(4.29:1 and 4.77:1 respectively). Ideally this should be a 7:1 contrast to meet [the WCAG AAA standard](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html).